content/en/blog/_posts/2022-12-6-provision-volumes-from-cross-namespace-snapshots-alpha.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: blog
+title: "Kubernetes 1.26: alpha support for provision volumes from cross-namespace snapshots"
+date: 2022-12-6
+slug: provision-volumes-from-cross-namespace-snapshots-redesigned
+---
+
+**Authors:**
+Takafumi Takahashi (Hitachi Vantara)
+
+Kubernetes v1.26, released earlier this month, introduced an enable usage of provision of persisten volume claim from volume snapshot in other namespaces.  
+Before Kubernetes v1.25, by using volume snapshots feature, users can provision volumes from snapshots. However, it only works for the `VolumeSnapshot` in the samme namespace, therefore users can't provision a persisten volume claim in one namespace from a `VolumeSnapshot` in other namespace. On the other hand, there are use cases that require to share the `VolumeSnapshot` across namespaces.  
+ To solve this problem,Kubernetes v1.26 includes a new API field called `dataSourceRef2`.
+
+## How it works
+
+Once the csi-provisioner finds a VolumeSnapshot is specified with non-empty namespace as dataSourceRef2, it checks all ReferenceGrants in　PersistentVolumeClaim.spec.dataSourceRef2.namespace to see if access to the snapshot is allowed. If it is allowed, the csi-provisioner provisions a volume from the snapshot.  
```

Comment:
We should recap what CSI is, with a hyperlink to read more.

I recommend mentioning / explaining what a ReferenceGrant is at this point, and then:

```suggestion
Once the csi-provisioner finds that a VolumeSnapshot is specified with a `dataSourceRef02` that
has a non-empty namespace name,
it checks all reference grants within the namespace that's specified by the `.spec.dataSourceRef2.namespace`
field of the PersistentVolumeClaim, in order to see if access to the snapshot is allowed.
If any ReferenceGrant allows access, the csi-provisioner provisions a volume from the snapshot.  
```